lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -216,6 +219,25 @@ export abstract class CacheManager implements ICacheManager {
         credential: ValidCredentialType
     ): string;
 
+    /**
+     * Read the user-supplied cache into memory, if applicable
+     * @param serializableCache 
+     * @param persistencePlugin 
+     */
+    async readUserDefinedCacheIntoMemory(
+        serializableCache: ISerializableTokenCache | null,
+        persistencePlugin: ICachePlugin | null
+    ): Promise<void> {
+        if (serializableCache && persistencePlugin) {
+            const cacheContext = new TokenCacheContext(
+                serializableCache,
+                true,
```

Comment:
Flag should be set to false, since the cache hasn't changed. It only changes when: 

- MSAL gets new tokens from AAD and merges them in
- Some `Remove` API is used.